java/springboot_hikaricp_using_dsql_connector/build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+application {
+    mainClass = 'org.example.Example'
+}
+
+group = 'org.example'
+version = '1.0-SNAPSHOT'
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal() // For Aurora DSQL JDBC Connector
+}
+
+dependencies {
+    // Aurora DSQL JDBC Connector (includes PostgreSQL driver)
+    implementation 'software.amazon.dsql:aurora-dsql-jdbc-connector:1.0.0'
+    
+    // HikariCP connection pooling
+    implementation 'com.zaxxer:HikariCP:5.1.0'
+    
+    // AWS SDK for DSQL (included in connector but explicit for clarity)
+    implementation 'software.amazon.awssdk:dsql:2.31.32'
+    
+    // Logging
+    implementation 'org.slf4j:slf4j-api:2.0.17'
+    implementation 'ch.qos.logback:logback-classic:1.4.14'
+    
+    // Test dependencies
+    testImplementation platform('org.junit:junit-bom:5.10.0')
+    testImplementation 'org.junit.jupiter:junit-jupiter'
+    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
+}
+
+tasks.test {
+    useJUnitPlatform()
```

Comment:
Are the unit test dependencies necessary?

Most of our other code samples include a test entry point, and some more complicated samples have proper unit tests that cover things like connection pool behavior.